### ISSUE TITLE
Deamon does not start after fresh install on Windows 10

### ISSUE THREAD

**Issue by gugadev** [typicode/hotel#212]
I need to use Windows for a while. After install hotel and run `hotel start` but it never starts. I can know this easily running `netstat -ano | Select-String "2000"` (no results).

Where is the *deamon.log* file in Windows?

Thanks.

**Comment by typicode** [comment]
Hi @gugadev,

In your user home, you should find a directory `./hotel`.
Feel free to paste the content of the `daemon.log`.

**Comment by gugadev** [comment]
Hi @typicode, thanks for reply. The folder exists but there is no any *daemon.log*, just *startup* and *conf.json* files.

**Comment by typicode** [comment]
I have Windows 10 too and last time I tested it, it worked.
Maybe there was an update that broke something.

I'll give another try.

Maybe it can help debugging, `hotel start` creates a `vbscript` in `AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup` then starts the vbscript to start hotel.
https://github.com/typicode/user-startup#how-it-works

**Comment by gugadev** [comment]
I'll do that, @typicode. I'll update this issue very soon. Thanks for help.

**Comment by typicode** [comment]
I've retried on Windows 10 with Node 8.1.0 and didn't have problems. I'll update Node.

**Comment by gugadev** [comment]
I ran the vb script with no errors but the *daemon.log* is not created.

**Comment by typicode** [comment]
You're welcome, happy to help (and hope a solution will be found). If you can find hotel module directory, you can run manually the daemon with the following command:

```sh
node lib/daemon
```

This way, you can excude the vbscript and you'll see directly the daemon output with perhaps a stacktrace.

I've not yet had feedback about rights issues on Windows, but some people said to have issues with Antivirus blocking the vbscript (not saying you should disable it ^^). I've tried with Windows Defender on my machine and there's no particular issue.

**Comment by gugadev** [comment]
Yup @typicode, running the daemon directly with node works as expected. Also, I added the *hotel.vbs* script to exclusions in Windows Defender, but it doesn't works.

**Comment by eimerreis** [comment]
Same for me, running the daemon directly works fine. As a workaround I'm running a bash script at startup of my computer, which runs the daemon directly

**Comment by GitHub-Mike** [comment]
@typicode:
> Maybe it can help debugging, `hotel start` creates a `vbscript` in `AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup` then starts the vbscript to start hotel.

I have the same problem and the reason is actually that the hotel.vbs file is not created in folder:

`%AppData%\Microsoft\Windows\Start Menu\Programs\Startup`
